src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="/"
+      class="search wrap_form"
+      data-qa="big"
+    >
+      <img
+        class="search__icon search__icon--big"
+        src="./images/Search.svg"
+        alt="search"
+      >
+      <input
+        data-qa="keypress"
+        class="search__input search__input--big"
```

Comment:
```suggestion
        class="search__icon"
        src="./images/Search.svg"
        alt="search"
      >
      <input
        data-qa="keypress"
        class="search__input"
```
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/e08f584a-4eea-4302-9733-23646aa16831)